requirements.yml

```diff
@@ -224,3 +224,7 @@
 
 - name: pytest-mpl
   version: 0.5
+
+- name: extinction
```

Comment:
This *should* be copied from conda-forge without any additional action on your part....the way the recipes are generated right now is:

+ Use a recipe template if there is one.
+ Copy the recipe from conda-forge if it exists.
+ Generate a recipe using `conda skeleton`

If it exists the package is on the conda-forge channel the binary gets copied into the astropy channel without building.